### ISSUE TITLE
Feature request: Add shortcuts

### ISSUE THREAD

**Issue by disharmonized** [paralect/robomongo#1251]
It is very frustrating, that robomongo does not support shortcuts for actions. Really, i want to delete collections or databases by pressing "Del" button, rename it by pressing 'F2' button, and so on. You should do that, guys. Please.

**Comment by simsekgokhan** [comment]
Hi @disharmonized , I agree that both suggestions will be very useful. We have added into our list. We are currently focused delivering some critical features Replica Sets and Export/Import asap. But we will still consider your suggestions since they are relatively easier and faster to implement.

**Comment by andreas-mausch** [comment]
:+1: 
The tool is very neat, but if I have to "right-click -> View Document" each time I want to see the JSON it's annoying af.
I prefer the tree mode for query results, but I really need an easy shortcut to view the JSON for a selected entry. Please.

**Comment by darrylsepeda** [comment]
I agree, it will be very helpful for developers if there is shortcuts to do delete record, edit record, etc.
Shortcut will make our life easier.
